api/src/main/java/com/bionicuniversity/edu/fashiontips/api/AuthController.java

```diff
@@ -0,0 +1,28 @@
+package com.bionicuniversity.edu.fashiontips.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ *  Controller which handles authenticate requests
+ */
+@CrossOrigin
+@RestController
+public class AuthController {
+
+    @RequestMapping(value = "/me", method = RequestMethod.GET, produces = MediaType.TEXT_PLAIN_VALUE)
```

Comment:
@RestController by default sends Header - MediaType.Json.
This method don't return Json, just simple string, so we must set it manually.